src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -275,10 +281,42 @@ export const BorrowInput = ({
     isEstimatedSweepFeesDataSuccess,
   ])
 
+  const useLendingQuoteQueryArgs = useMemo(
+    () => ({
+      collateralAssetId,
+      collateralAccountId,
+      borrowAccountId,
+      borrowAssetId: borrowAsset?.assetId ?? '',
+      depositAmountCryptoPrecision: depositAmountCryptoPrecision ?? '0',
+    }),
+    [
+      borrowAccountId,
+      borrowAsset?.assetId,
+      collateralAccountId,
+      collateralAssetId,
+      depositAmountCryptoPrecision,
+    ],
+  )
+  const {
+    data,
+    isLoading: isLendingQuoteLoading,
+    isRefetching: isLendingQuoteRefetching,
+    isSuccess: isLendingQuoteSuccess,
+    isError: isLendingQuoteError,
+    error: lendingQuoteError,
+  } = useLendingQuoteOpenQuery(useLendingQuoteQueryArgs)
+
+  const lendingQuoteData = isLendingQuoteError ? null : data
+
+  useEffect(() => {
+    setConfirmedQuote(lendingQuoteData ?? null)
```

Comment:
Perhaps, or perhaps not? Big fan of RTK/reselect but In the case of lending, I believe there's nothing wrong with drilling props down and passing down setters, it's not an utterly complex domain and we really don't rely on the values anywhere else in the app, which doesn't warrant a `global` state since the data isn't shared globally?